.github/workflows/ci.yaml

```diff
@@ -0,0 +1,185 @@
+name: EC2 Metadata Mock CI and Release
+
+on: [push, pull_request, workflow_dispatch]
```

Comment:
Still doing some reading, but is `workflow_dispatch` here so we can kick off these jobs from the repo without any changes (i.e. commits, prs, tags, etc.)?

Reply:
yep, for manual kick off